kms/protobuf/request.proto

```diff
@@ -233,6 +233,9 @@ message CreateIdentityRequest {
 
   // IsServiceAccount indicates whether the identity is a service account.
   bool IsServiceAccount = 3 [ json_name = "service_account" ];
+
+  // Tags are optional metadata labels attached to the identity as key-value pairs.
+  map<string,string> Tags = 4 [ json_name="tags" ];
```

Comment:
Shouldn't this be marked as `optional`?

Reply:
They are automatically "optional" in proto3.
> If no explicit field label is applied, the default field label, called “implicit field presence,” is assumed. (You cannot explicitly set a field to this state.) A well-formed message can have zero or one of this field (but not more than one). You also cannot determine whether a field of this type was parsed from the wire. An implicit presence field will be serialized to the wire unless it is the default value.